lib/components/command_bar.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:provider/provider.dart';
+import 'package:flutter/material.dart';
+import 'package:rtchat/models/commands.dart';
+import 'package:rtchat/models/channels.dart';
+import 'package:rtchat/models/adapters/actions.dart';
+
+class CommandBarWidget extends StatelessWidget {
+  final void Function() chatInputUnfocus;
+  static const _commandBarHeight = 55.0;
+
+  const CommandBarWidget({Key? key, required this.chatInputUnfocus})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
```

Comment:
Does this not autosize? ie the `SizedBox` shouldn't be necessary because the button bar has intrinsic size?